### PR TITLE
Add faucet token

### PR DESCRIPTION
**Pull request by voyager-gold**
- Add faucet token and deploy scripts

### CODE REVIEW

File: tasks/dev/7_faucet_tokens.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+task('dev:deploy-faucet-tokens', 'Deploy faucet tokens for dev enviroment').setAction(
+  async ({ verify }, localBRE) => {
+    await localBRE.run('set-DRE');
+
+    const faucetUSDC = await new FaucetERC20Factory(await getFirstSigner()).deploy(
+      'USDC',
+      'USDC',
+      6,
+      BigNumber.from(10).pow(6),
```
Comment (Hsiang-xxs) [inline tasks/dev/7_faucet_tokens.ts:14]: `2` stablecoins per request

File: tasks/dev/7_faucet_tokens.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      3600 * 24
+    );
+
+    const faucetUSDT = await new FaucetERC20Factory(await getFirstSigner()).deploy(
+      'USDT',
```
Comment (Hsiang-xxs) [inline tasks/dev/7_faucet_tokens.ts:19]: correct the `name` for stablecoin

File: tasks/dev/7_faucet_tokens.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      'USDT',
+      6,
+      BigNumber.from(10).pow(6),
+      3600 * 24
+    );
+
+    const faucetDAI = await new FaucetERC20Factory(await getFirstSigner()).deploy(
+      'DAI',
```
Comment (Hsiang-xxs) [inline tasks/dev/7_faucet_tokens.ts:27]: correct the `name` for stablecoin

**Review by Hsiang-xxs — CHANGES_REQUESTED** [review]
Fix all issues

File: contracts/mocks/tokens/FaucetERC20.sol [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+  constructor(
+    string memory name,
+    string memory symbol,
+    uint8 decimals,
+    uint256 _maxAmountPerMint,
+    uint256 _minMintFrequency
+  ) public ERC20(name, symbol) Ownable() {
+    _setupDecimals(decimals);
+
```
[the remark this answers is not in the document]
Comment (voyager-gold) [reply contracts/mocks/tokens/FaucetERC20.sol:26]: Isn't `msg.sender` owner in Ownable constructor? @Hsiang-xxs

[the remark this answers is not in the document]
Comment (Hsiang-xxs) [reply contracts/mocks/tokens/FaucetERC20.sol:26]: I thought I already resolved this convo

**Pull request merged**